### ISSUE TITLE
Move the staking banner from asset page to portfolio

### ISSUE THREAD

**Issue by fl-y** [EmerisHQ/demeris#1227]
From @nassdonald [slack thread](https://allinbits.slack.com/archives/C01JJ28Q2G5/p1646337280047649?thread_ts=1646292871.060829&cid=C01JJ28Q2G5)

**Why**

> I believe this may have been a detail missed in the design -> dev handoff. CC [@Benjamin](https://allinbits.slack.com/team/U01UGTN5LCT)
> To clarify, we have a space in the UI for banners under the swap widget on both the Portfolio page and the Asset Detail page.
> On the Asset Detail page it doesn’t make sense to show this banner and the staking CTA already provided in the empty state of the Staking section. It also doesn’t make sense to show the banner on this page when a user is already staking that asset – the feature will be discoverable enough where it is.
> But on the Portfolio page, it’s important that we have discoverability of the new Staking feature. This is a little more tricky from a UX perspective without the new Yield page we’re working on…
> The problem is that it would be weird to show a banner for staking ATOM on the Portfolio page if you A) don’t hold ATOM or B) you are interested in staking other assets. When we introduced the Moonpay banner, it made sense to be ATOM-focused because that was the asset required to pay fees for swaps etc. But staking applies to many assets so it feels weird to use “Stake ATOM” as the CTA on the Portfolio page.

**What**

> TL;DR: Let’s remove the banner from Asset Detail page as it’s repetitive. If we can make the banner dynamic based on which staking assets you have available, we can show it on the Portfolio page with the staking asset you hold the most of, perhaps.
